crates/bevy_render/src/texture/fallback_image.rs

```diff
@@ -20,8 +20,15 @@ use crate::{
 ///
 /// Defaults to a 1x1 fully opaque white texture, (1.0, 1.0, 1.0, 1.0) which makes multiplying
 /// it with other colors a no-op.
-#[derive(Resource, Deref)]
-pub struct FallbackImage(GpuImage);
+#[derive(Resource)]
+pub struct FallbackImage {
+    pub d1: GpuImage,
```

Comment:
Doc strings for all of these fields please: they're quite challenging to follow.